test/e2e/swaps/swaps-notifications.spec.js

```diff
@@ -5,10 +5,58 @@ const { withFixtures } = require('../helpers');
 const { withFixturesOptions, loadSwaps, buildQuote } = require('./shared');
 
 describe('Swaps - notifications', function () {
+  async function mockTradesApiPriceSlippageError(mockServer) {
```

Comment:
Would this mock be useful for other swaps tests as well? Surely at least some of the others must hit the `/trades` endpoint as well

Reply:
Oh, I see, this is the only one that uses this specific trade. So this mock is only applicable here.